apps/api/src/database/schema.prisma

```diff
@@ -570,12 +570,12 @@ model RepresentationAction {
   id                   Int            @id @default(autoincrement())
   representationId     Int
   type                 String? // STATUS | REDACTION | REDACT_STATUS
-  status               String?
+  status               String? // AWAITING_REVIEW | REFERRED | INVALID | VALID | WITHDRAWN
   previousStatus       String?
   redactStatus         Boolean?
   previousRedactStatus Boolean?
-  invalidReason        String?
-  referredTo           String?
+  invalidReason        String? // DUPLICATE | MERGED | NOT_RELEVANT | RESUBMITTED
```

Comment:
This is a false representation of the data stored and does not align with the validators which have the allowable strings. 

These are misleading and should be removed.

Reply:
oops, those were just an example for my other comment, removed

Reply:
or rather, I'll replace with correct values

Reply:
delete them.